data/battle_anim_scripts.s

```diff
@@ -3450,26 +3450,26 @@ Move_UPROAR: @ 81CBEFC
 
 Move_HEAT_WAVE: @ 81CBFC6
 	loadspritegfx 10261
-	createvisualtask sub_80E2C60, 5, 10261, 0, 6, 6, 31
+	createvisualtask AnimTask_BlendSpriteColor, 5, 10261, 0, 6, 6, 31
 	createvisualtask do_boulder_dust, 5, 1
-	createvisualtask sub_80D6080, 6, 6, 31
+	createvisualtask AnimTask_BlendInterfaceColor, 6, 6, 31
 	panse_1B SE_W257, 192, 63, 2, 0
 	delay 4
 	createvisualtask sub_80D5DDC, 5
 	delay 12
-	createsprite gBattleAnimSpriteTemplate_83DACE8, 40, 10, 2304, 96, 1
+	createsprite gSandstormDustSpriteTemplate, 40, 10, 2304, 96, 1
```

Comment:
probably shouldn't be named `gSandstormDust` is it's used for things like Heat Wave and Mud Sport